### PR TITLE
rg-35: add 404 page. Dependency: rg-3

### PR DESCRIPTION
**Pull request by MaxPoplavskiy**
Create 404 page:
image

### CODE REVIEW

**Comment by nickolay-grechkin** [comment]
As I understand it correctly, you created your branch from [this one](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/pull/5/files). In that case, you need to create a merge request not to the dev branch, but to the branch from which you created yours. Your pull request will be merged after the original one, and reviewers won't need to check the same changes twice. And also resolve conflicts, please.

**Review by nickolay-grechkin — CHANGES_REQUESTED** [review]
As I understand it correctly, you created your branch from [this one](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-resumegemm/pull/5/files). In that case, you need to create a merge request not to the dev branch, but to the branch from which you created yours. Your pull request will be merged after the original one, and reviewers won't need to check the same changes twice. And also resolve conflicts, please.

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

File: frontend/src/app/app.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const App: React.FC = () => {
                     <Link to={AppRoute.SIGN_IN}>Sign in</Link>
                 </li>
                 <li>
-                    <Link to={AppRoute.SIGN_UP}>Sign up</Link>
+                    <Link to={AppRoute.SIGN_UP}>Sign up</Link>11
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/app/app.tsx:39]: Looks like this change is redundant

Comment (MaxPoplavskiy) [reply frontend/src/app/app.tsx]: Done. Remove redundant text in app

**Review by apofis96 — APPROVED** [review]

**Pull request merged**